oci/oci.go

```diff
@@ -500,18 +513,20 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 			if err != nil {
 				return fmt.Errorf("status code conversion failed: %v", err)
 			}
-			c.state.ExitCode = int32(statusCode)
+			c.State.ExitCode = int32(statusCode)
 		}
 	}
 
+	c.toDisk()
```

Comment:
@sameo what I'm really doing here is storing `c.State` which contains fields like `Started/Finished/Stopped` - I guess we can move them directly in `config.json` though that's not updated at every state change. Also, afaict, docker does the same with its containers state - are they wasting so much IO as well?